docs/docker_deployment_guide.md

```diff
@@ -42,19 +42,26 @@ git clone https://github.com/MbinOrg/mbin.git
 cd mbin
 ```
 
-Build the Docker image:
+### Prepare and build Docker image
 
 > **Note**
 > If you're using a version of Docker Engine earlier than 23.0, run `export DOCKER_BUILDKIT=1`, prior to building the image. This does not apply to users running Docker Desktop. More info can be found [here](https://docs.docker.com/build/buildkit/#getting-started)
 
+1. First go to the _docker directory_:
+
 ```bash
-docker build -t mbin -f docker/Dockerfile .
+cd docker
 ```
 
-Create config files and storage directories:
+2. Build the docker image (from the root-directory of the repository):
+
+```bash
+docker build --no-cache -t mbin -f Dockerfile  ..
```

Comment:
I updated the command, so the user can stay in the `docker` directory. Also I added `--no-cache` so it shouldn't use existing cached docker layers to build the image.